src/mqueryfront/src/QueryField.js

```diff
@@ -206,14 +217,8 @@ class QueryField extends Component {
                         </div>
                     </div>
                 </div>
-                <div className="form-group">
-                    <textarea
-                        name="rawYara"
-                        className="form-control mquery-yara-input"
-                        onChange={this.handleInputChange}
-                        readOnly={this.state.isLocked}
-                        value={this.state.rawYara}
-                    />
+                <div className="mt-1 monaco-container">
+                    <QueryMonaco ref="editor" />
```

Comment:
String Refs are legacy (https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs). Maybe we should use `React.createRef()`?